ext/node/resolution.rs

```diff
@@ -548,7 +548,9 @@ fn parse_package_name(
   } else if specifier.starts_with('@') {
     is_scoped = true;
     if let Some(index) = separator_index {
-      separator_index = specifier[index + 1..].find('/');
+      separator_index = specifier[index + 1..]
+        .find('/')
+        .map(|new_index| index + 1 + new_index);
```

Comment:
Oops. +1 for unit tests for this.

Reply:
Appears to be a copy-paste from `node_resolver` crate, so not your fault :P